easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.5a1-CrayPGI-19.10-cuda-10.1.eb

```diff
@@ -10,7 +10,11 @@ homepage = 'http://www.quantum-espresso.org/'
 description = """Quantum ESPRESSO is an integrated suite of computer codes
```

Comment:
```python
description = """Quantum ESPRESSO is an integrated suite of computer codes for
electronic-structure calculations and materials modeling at the nanoscale. It
is based on density-functional theory, plane waves, and pseudopotentials (both
norm-conserving and ultrasoft). PGI Fortran gives the warning "ieee_inexact
FORTRAN STOP" with QE GPU.  Set NO_STOP_MESSAGES=1 in your batch script to
avoid the warning.
More details on the PGI User Forum at https://www.pgroup.com/userforum
"""
```

Reply:
Thanks for the review @jgphpc: I wished to separate the last part of the `module help` command from the description of the code. Have a look at the output on screen with the following commands on Piz Daint:
```
module use  /apps/daint/UES/sandbox/lucamar/easybuild/modules/all
module help QuantumESPRESSO/6.5a1-CrayPGI-19.10-cuda-10.1
```

Reply:
What about the following output on screen?
```
Quantum ESPRESSO is an integrated suite of computer codes for
electronic-structure calculations and materials modeling at the nanoscale. It
is based on density-functional theory, plane waves, and pseudopotentials (both
norm-conserving and ultrasoft). 

Note: PGI Fortran gives the warning "ieee_inexact FORTRAN STOP" with QE GPU.  
Set NO_STOP_MESSAGES=1 in your batch script to avoid the warning.
More details on the PGI User Forum at https://www.pgroup.com/userforum
```

Reply:
Thanks @jgphpc, see my comment in line